API/CLI.md

```diff
@@ -67,14 +67,15 @@ Along with the package.json and index.js included in the `t2 init` process, ther
 * `.tesselinclude` the overriding and opposite behavior of `.tesselignore`, this file should list any files or directories you want included with the T2 bundling and deployment process. This is handy for including non-JavaScript assets, like HTML, CSS, and images, for use within your project.
   * In your JavaScript program, assets listed in the `.tesselinclude` file should always be accessed using `__dirname` (read more about `__dirname` in the [Node.js docs](https://nodejs.org/api/globals.html#globals_dirname)). For example, if there is a file at `public/index.html` and it's listed in your `.tesselinclude`, from your JavaScript program, you'd read it like this: `fs.readFile(path.join(__dirname, "public/index.html"), "utf8", (error, contents) => ...);`.
 
-### Tessel Management  
-* `t2 provision` will authorize your computer to access a Tessel over SSH. (USB-connected Tessel only)
+### Tessel Management
+* `t2 key` will generate a local SSH keypair for authenticating to a Tessel.  Use with the `generate` argument to generate a new local SSH keypair for authenticating to a Tessel.
 * `t2 list` will show what Tessels are available over WiFi and USB.
+* `t2 provision` will authorize your computer to access a Tessel over SSH. (USB-connected Tessel only)
 * `t2 rename` will change the name of a Tessel.
-* `t2 reboot` will reboot your Tessel.  
-* `t2 key` will generate a local SSH keypair for authenticating to a Tessel.  Use with the `generate` argument to generate a new local SSH keypair for authenticating to a Tessel.
+* `t2 reboot` will reboot your Tessel.
+* `t2 restore` Restore your Tessel to factory defaults by installing the factory version of [OpenWrt](https://openwrt.org/).
```

Comment:
It would be more accurate to say it installs the factory version of [`openwrt-tessel`](https://github.com/tessel/openwrt-tessel) and [`t2-firmware`](https://github.com/tessel/t2-firmware).